src/tls13.c

```diff
@@ -9448,15 +9451,24 @@ int wolfSSL_accept_TLSv13(WOLFSSL* ssl)
 
         case TLS13_ACCEPT_FINISHED_DONE :
 #ifdef HAVE_SESSION_TICKET
-    #ifdef WOLFSSL_TLS13_TICKET_BEFORE_FINISHED
-            if (!ssl->options.verifyPeer) {
-            }
-            else
-    #endif
-            if (!ssl->options.noTicketTls13 && ssl->ctx->ticketEncCb != NULL) {
-                if ((ssl->error = SendTls13NewSessionTicket(ssl)) != 0) {
-                    WOLFSSL_ERROR(ssl->error);
-                    return WOLFSSL_FATAL_ERROR;
+            for (ticketsSent = 0; ticketsSent < ssl->options.maxTicketTls13;
+                    ticketsSent++) {
```

Comment:
Why is a for loop needed here? I don't see any added value for it where a single `if` expression will do. I don't see `ticketsSent` used?

Reply:
By default OpenSSL sends 2, have set our default to 1 to try and match previous behavior. Each call to SendTls13NewSessionTicket in the for loop is sending out a new session, being called up to ssl->options.maxTicketTls13 times.

Reply:
To me it does not make sense for us to send more than one ever. I would really like to hear @SparkiDev opinion on this.

Openssl did it as a workaround to an issue with old session resumption callback behavior. The openssl change https://github.com/openssl/openssl/commit/36ff232cf2bf5dfcaf9e60a8c492439428a243bb

If we are interoperating with openssl we should handle receiving two session tickets without an error. If openssl is expecting us to send two then I suppose this support is merited. Have you done any interop testing for these cases?